NEWS.md

```diff
@@ -1,5 +1,6 @@
 ## 2020-12-07 v4.2.0-SNAPSHOT
 * [MODSOURMAN-385](https://issues.folio.org/browse/MODSOURMAN-385) Enable OCLC update processing.
+* [MODSOURCE-212](https://issues.folio.org/browse/MODSOURCE-212) Fix matching by 999 ff s field
```

Comment:
did you mean 217?

Reply:
@KaterynaSenchenko Yes, sure. Thank you!